src/main/java/core/basesyntax/MainApp.java

```diff
@@ -12,5 +12,15 @@
  * - Отправить все машини на работу и остановить работу всех машин.
  */
 public class MainApp {
-
+    public static void main(String[] args) {
+        Excavator excavator = new Excavator();
```

Comment:
What will happen if we write:
Machine excavator = new Excavator();
and the same with other objects?